mage_ai/data_preparation/repo_manager.py

```diff
@@ -1,12 +1,24 @@
 from mage_ai.data_preparation.templates.utils import copy_templates
 import os
 
-
 def init_repo(repo_path: str) -> None:
     """
     Initialize a repository under the current path.
     """
+
     if os.path.exists(repo_path):
         return
+    else:
```

Comment:
You might not need to add these since I already added the code to copy template files. You can add additional files to this folder if needed: https://github.com/mage-ai/mage-ai/tree/master/mage_ai/data_preparation/templates/repo

Reply:
oh i see